regression/ansi-c/goto_convert/switch_range_bounds/test.desc

```diff
@@ -0,0 +1,8 @@
+CORE
```

Comment:
Isn't this one GCC-only?

Reply:
No, both `clang` and `gcc` report the same error.

Reply:
Ok, but Visual Studio might not? But either way this one should fail in the front-end, so it's probably ok.